lib/constants.js

```diff
@@ -201,5 +201,15 @@ module.exports = Object.freeze({
     INACTIVE: 'inactive',
     DEPRECATED: 'deprecated'
   },
-  BASE64_ENC_NULL: 'bnVsbA=='
+  BASE64_ENC_NULL: 'bnVsbA==',
```

Comment:
where is this constant used?

Reply:
Not in this implementation. Was added as part of this pr.
https://github.com/cloudfoundry-incubator/service-fabrik-broker/commit/10a04aaf6db845f589f87b944fe9c30f26cfcdd4#diff-3b1de79a454278043d7b0e4887934dedR204